project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss

```diff
@@ -25,3 +25,33 @@
 }
 
 
+h1 {
+  font-size: 3.2rem;
+  margin: 2rem 0 2.7rem;
+  line-height: 3.9rem;
+}
+
+#search-facets {
+  width: 30%;
+}
+
+.sidebar-first .main-content {
+  width: 65%;
+}
+
+.current-search-item-results-count {
+  margin-top: 0;
+}
+
+#block-exposedformpublications-searchpublication-search-page {
```

Comment:
Not sure you need lines 46-57 as all these sizes come from the nicsdru_unity_theme. You may need to update nicsdru_unity_theme in your local machine to see these changes.

Reply:
Thanks , I have removed these lines